### ISSUE TITLE
Answer replies versus quoting replies

### ISSUE THREAD

**Issue by gryphonshafer** [gryphonshafer/Quizzing-Rule-Book#111]
Consider Matt 8:20:

> Jesus replied, "Foxes have dens and birds have nests, but the Son of Man has no place to lay his head."

Let's say there's an MA in the form:

> What have nests?

The quizzer responds:

> Foxes have dens and birds have nests.

This illustrates a fundamental problem with the rules as they're currently written. If a quizzer responds with the above, they are de facto correct (in that essentially no quizmaster is going to count them incorrect); however, they are de jure incorrect. Technically, the quizzer's first answer is "foxes," which is incorrect.

Here are a couple reasons why this is true:

- The rule book has no concept of "quoting" versus "answering" -- all responses are answers
- The rule book has no concept of a quizzer pausing or not mid-response -- it's irrelevant if the quizzer saying "foxes... have dens and birds have nests" includes a pause of half a second or 20 seconds (or 0 seconds)

However, I would expect very close to 100% of quizmasters would rule the quizzer correct. I would further expect very close to 100% of coaches would consider this ruling correct. This illuminates a rule book bug that must be fixed.

*It would be tempting to think this can be easily solved by establishing a "quizzer is quoting versus answering" concept in the rule book.* However, practically speaking, how could this be determined? How can a quizmaster know objectively if a quizzer is quoting or answering? If a quizzer begins with "foxes" and pauses for a few seconds, the quizmaster might count them incorrect, and there would then follow a challenge that the quizzer was quoting and paused instead of answering. The only option is for the quizmaster to wait for the full 30 seconds, which in an of itself isn't a horrible outcome; however, an ill-prepared quizzer could game this scenario by answering "foxes" and pausing to see if they're counted correct, then continuing.

*It would be tempting to think this can be easily solved by considering the length of pause between "foxes" and the rest of the the response.* However, what's the objective length of a pause that causes the response to be an answer versus a quote with a pause inserted for recall purposes? How would that be objectively enforced consistently between quizmasters?

This is a big topic with lots of discussion necessary before we can be sure we're going to get this right. Nothing within this initial write-up should be construed as suggesting an easy solution. I personally believe there's an objective solution, but it will take a fair bit of thought, debate, and work to identify it.

**Comment by jswingle** [comment]
I presume that your example MA "What have nests?" is actually intended to be an example INT instead.

I am of the opinion that there's no need for a rule about this, because I've never personally had an experience where it was difficult to rule on this sort of thing. I'd be curious to hear from anyone else if they've encountered a situation like this example where a rule would have helped them make a more objective ruling.

When QM/AJing, if a quizzer jumped on this example question and said "Foxes" and didn't immediately follow it up with "have dens," I would rule them incorrect. Admittedly, what constitutes "immediately" is a bit subjective, but it's impossible for the rulebook to make every little detail perfectly objective. I think in the overwhelming majority of scenarios, a QM/AJ is going to be able to tell when a quizzer knows the answer and when they're just trying to game it with "oh, I was just quoting!" technicalities.

All that being said, I do have an idea for a rule which would probably solve this issue, but I'm not sure if it would create any other negative consequences: a rule which requires the quizzer to _repeat_ all the content of the question, even content already given by the quizmaster at the time of the jump. Thoughts?

**Comment by gryphonshafer** [comment]
You're right about it not being an MA. Silly typo from me. It's an INT.

That said: Whether you've found it difficult isn't relevant. The point is it requires a subjective interpretation. There's no objective way to evaluate "didn't immediately follow it up with". And even if there was, it doesn't solve the underlying problem: There's no rule book difference between providing an explicit answer and quoting.

Technically, saying, "Foxes have dens and birds have nests" is incorrect. It doesn't correctly answer the question: "What have nests?" Like I wrote, essentially no quizmaster is going to count the quizzer incorrect, which I think is what we want the outcome to be. But I don't see how we get there with the current rule book as written. It's just a tribal custom right now.

Requiring the quizzer to repeat is an interesting idea and worth exploring for others reasons, but I don't think it would solve the problem in this example.

**Comment by jswingle** [comment]
Further explanation on why this would solve the problem: If the quizzer had to give the material found in the question again, they wouldn't be able to game it by jumping, saying "Foxes" and then pausing. They would have to say an entire phrase, "Foxes have nests" - at which point they would be counted incorrect for giving wrong information.

**Comment by kclimenhaga** [comment]
I have also never found this to be difficult to rule on. In terms of "immediately", for me, if I've been able to process that the quizzer has given an answer, and the answer is wrong, before the quizzer has continued talking, that constitutes an answer instead of quoting. Intonation also helps; people speak differently when providing an answer than when quoting. 

I'm hesitant to require quizzers to repeat the information in the question. I know quizzers would be capable of learning to do that, but I don't really see value to it. Maybe someone can provide more insight?

**Comment by jttower** [comment]
I don't think we need to make a rule about this.  QMs and AJs have to make judgment calls on a regular basis (are these synonyms, for example). We don't want to "punish" a quizzer for quoting the context, because some quizzers think that way (in phrases). I think a QM/AJ can tell when a quizzer is quoting the verse or slowly trying to "fish" for the answer.  I can think of examples when a quizzer too quickly gave a one-word answer to a question, it was clearly the wrong answer and they knew immediately afterward that it was the wrong answer.  You can tell when a quizzer is confidently stating their answer (as a quote or otherwise) or when they are unsure and "fishing".  

If there is any doubt or question as to how the quizzer answered, that is why the AJ and QM can discuss and make a ruling.  And if there was doubt from a quizzer's perspective, they are able to challenge and give input.  I think we have safeguards built into the rule book for situations like this.  

We aren't going to be able to make objective rules for every situation.  If we over-regulate and make a bunch of complicated rules in an effort to make everything completely objective, the rule book will be over-complicated and cumbersome, which could deter people from quizzing.  We want to make quizzing more accessible to people, not more complicated.  I know that some will say that having a bunch of objective rules will make it easier to understand, but I think the more rules there are to keep track of, the more confusing quizzing can seem to some.

**Comment by josiah-leinbach** [comment]
I think this is an instance where we ought apply the "I can't define it [i.e. quoting vs. answering] but I know it when I see/hear it." To clarify that in the rulebook could be as simple as an addendum saying, "It will be left to the judgment of QMs and AJs to determine whether quizzer was quoting the context or offering a definitive answer."

**Comment by jswingle** [comment]
It's worth adding to the discussion that, presently, there is no rulebook language distinguishing "giving an answer" vs. "quoting." It's something us quizzing enthusiasts commonly talk about, but not in the rulebook at all. Strictly and technically speaking, anything said by the quizzer would be "giving an answer."

But in practice, we don't go by this "strict and technical" interpretation of the rulebook, because human communication is far too complex to encapsulate in a rulebook. In addition to the various ways in which a quizzer could say "Foxes" in the example above, consider a different scenario: a quizzer mumbling some verses to themselves while thinking through the answer. If the quizzer is clearly not trying to say things to the QM, but the QM happens to _barely hear_ the quizzer say something out of context under their breath, should we count them wrong for that? I would say no. But more importantly, I would _also_ say that there doesn't need to be a rule about that. It's a common-sense application of the rulebook to the unspoken rules of human communication. 

Of course, we could get around this issue by turning quizzing into the Bible equivalent of a spelling bee, and requiring quizzers to say upon jumping, "John 11:35. Jesus wept. John 11:35." This would make it easier to define objectively, but would also make quizzing lame. Therefore, if we don't want quizzing to be lame, we need to give some latitude to the officials to judge the various details of each ruling.

We could add a new rule that defines "quoting," but I think it would be VERY tough to write such a rule in a way that doesn't create a hundred additional points of confusion. Just a hunch there. 

My overall conclusion is that there don't need to be rule changes here and @josiah-leinbach's rationale is the correct one: "I can't define it but I know it when I see it."

**Comment by ARMediting** [comment]
In Central District, in either some years or some divisions, a rule was made to allow a quick correction of a blurted-out answer. Along the lines of "Peter--no, Andrew!" This is the part of the first-answer rule that's painful to rule on. You could say an IBQ quizzer should know better and think before speaking. And of course there's still the question of how long is too long of a pause.

**Comment by jttower** [comment]
I don't think there should be an allowance for a quick correction of a blurted-out answer.  It is much easier to say that we need to take the first answer, otherwise it seems like you would have to say "is that your final answer?" or they could quick-correct themselves several times and then claim that the QM should have stopped them when they were correct. If an individual district wants to modify that rule on their own, that's up to them, but I definitely don't think we should even suggest it in the rule book.

In response to @jswingle 's scenario ("a quizzer mumbling some verses to themselves..."), one of my quizzers has brought this up to me in the past, that she almost hopes for someone to challenge her on this someday so that she can say, "You shouldn't be eavesdropping on my personal conversation with myself!"  :-)  Ha, ha!  

All joking aside, I agree that a quizzer could use mumbling as fishing for an answer, but again I think it falls under "I know it when I see it".  I like the suggestion from @josiah-leinbach of adding that one sentence leaving it up to QMs/AJs.  But again, there are already ways built into the rule book for giving discretion to the QM/AJ and allowing challenges from quizzers.

**Comment by jswingle** [comment]
I think the blurted-out answer rule could make a lot of sense as a district-specific addition to the rulebook, but yeah, would probably prefer being a bit "harsher" with the main rulebook.

**Comment by scottpeterson** [comment]
To frame it differently, it's not a desire to be "harsher." It's responding to the reality that the QM judging the "intent" of the quizzer will always be inconsistent and error-prone.

**Comment by gryphonshafer** [comment]
And to add a little to the frame: Inconsistent and/or error-prone is unfair, and unfair is anti-mission.

**Comment by kclimenhaga** [comment]
I agree that quizzers should not be allowed to correct a blurted out answer. If the answer is short enough to blurt out and correct, then they can learn to take five seconds to think it over. Regarding quoting vs. answering, I wouldn't want to punish quizzers for going back and quoting the verse an answer is found in rather than just saying the part of the verse that answers the question. Is that causing problems in other districts?
